src/sentry/static/sentry/app/views/organizationDiscover/result/pagination.jsx

```diff
@@ -11,33 +11,53 @@ export default class Pagination extends React.Component {
     getPreviousPage: PropTypes.func.isRequired,
     previous: PropTypes.object,
     next: PropTypes.object,
+    pageLimit: PropTypes.number.isRequired,
   };
 
+  getPageNumber(pageLimit) {
+    const {next} = this.props;
+    const pageNumber = next.cursor.split(':')[1];
+
+    return (
+      <PageNumber>
+        Results {pageNumber - pageLimit + 1} - {pageNumber}
```

Comment:
Should this state the actual number of results returned rather than the limit provided? For example, even if there are no results this is still going to read for example `Results 1 - 1000`.

Reply:
Yeah you're right, thank you!